File: DoohAdManager.Model/Application/INV/ScreenOperatingHour.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DoohAdManager.Model.Application.INV
{
    public class ScreenOperatingHour
    {
        public int Id { get; set; }
        public int ScreenId { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public DayOfWeek DayOfWeek { get; set; }
        public int AvgAudienceCount { get; set; }
        public DateTime CreatedAt { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public int? UpdatedBy { get; set; }
        public bool IsDeleted { get; set; }
        public DateTime? DeletedAt { get; set; }
        public int? DeletedBy { get; set; }
    }

    public class ScreenOperatingHourInsert
    {
        public int ScreenId { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public DayOfWeek DayOfWeek { get; set; }
        public int AvgAudienceCount { get; set; }
        public int? CreatedBy { get; set; }

    }
    public class ScreenOperatingHourDelete
    {
        public int Id { get; set; }
        public int? DeletedBy { get; set; }
    }

}

File: dooh-ad-manager/src/app/Application/INV/Screen/Models/screen-operating-hour.ts
import { DayOfWeek } from "../../../../Shared/Models/enum.model";

export interface MvScreenOperatingHour{
    id : number;
    screenId : number;
    startTime : string;
    endTime : string;
    dayOfWeek : DayOfWeek;
    avgAudienceCount : number;
    createdAt? : Date;
    createdBy? : number;
    updatedAt? : Date;
    updatedBy? : number;
    isDeleted? : boolean;
    deletedAt? : Date;
    deletedBy?: number;
}

export class MvScreenOperatingHourAdd {
    screenId : number;
    startTime : string;
    endTime: string;
    dayOfWeek : DayOfWeek;
    avgAudienceCount : number;
    createdBy? : number;

    constructor() {
        this.screenId = 0;
        this.startTime = '';
        this.endTime = '';
        this.dayOfWeek = DayOfWeek.Everyday;
        this.avgAudienceCount = 1;
    }
}

export interface MvScreenOperatingHourDelete {
    id: number;
    deletedBy?: number;
}